api/v1alpha7/types.go

```diff
@@ -197,6 +197,15 @@ type Network struct {
 	APIServerLoadBalancer *LoadBalancer `json:"apiServerLoadBalancer,omitempty"`
 }
 
+// NetworkStatus contains basic information about an existing neutron network.
+type NetworkStatus struct {
```

Comment:
Is it really a `Status`? Feels more like a `Spec` to me.

Reply:
It's definitely a Status: it has an ID apart from anything else. In the Spec we use NetworkFilter.